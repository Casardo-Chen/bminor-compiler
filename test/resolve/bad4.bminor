main: function void () = {
    x = 5; // x used before it is declared
    x: integer ;
    return 0;

}