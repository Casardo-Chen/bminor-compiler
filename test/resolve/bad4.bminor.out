resolve error: x is not defined.
1 resolve errors found.
